clickable.yaml

```diff
@@ -4,13 +4,9 @@ kill: berrylan
 build_args: CONFIG+=ubports
 
 dependencies_target: [
-    libavahi-client-dev,
-    libavahi-common-dev,
     qt5-default,
     libconnectivity-qt1-dev,
     libunity-api-dev,
```

Comment:
actually, also libunity-api-dev shouldn't be needed in here. nymea:app uses it for push notifications, but BerryLan does not support push notifications anyways.